mappings/net/minecraft/entity/mob/PiglinBrain.mapping

```diff
@@ -41,8 +41,9 @@ CLASS net/minecraft/class_4838 net/minecraft/entity/mob/PiglinBrain
 	METHOD method_24732 create (Lnet/minecraft/class_4836;Lnet/minecraft/class_4095;)Lnet/minecraft/class_4095;
 		ARG 0 piglin
 		ARG 1 brain
-	METHOD method_24733 onGuardedBlockBroken (Lnet/minecraft/class_1657;Z)V
+	METHOD method_24733 onGuardedBlockInteraction (Lnet/minecraft/class_1657;Z)V
```

Comment:
Do we have a convention on how to name `on___` methods? Should they be nouns (`onBlockBreak`, `onBlockInteraction`) or verbs (`onBlockBroken`, `onBlockInteracted`)?

Reply:
`onBlockInteracted` is better, will change

Reply:
This should probably also be added to conventions.md (assuming everyone agrees with verbs).